manifests/p/ProjectJupyter/JupyterLab-Desktop/3.5.0-1/ProjectJupyter.JupyterLab-Desktop.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ProjectJupyter.JupyterLab-Desktop
+PackageVersion: 3.5.0-1
+PackageLocale: en-US
+Publisher: Project Jupyter
+PackageName: JupyterLab-Desktop
+License: BSD-3-Clause license
+Copyright: Copyright © 2022 Project Jupyter

```

```suggestion
Copyright: Copyright © 2022 Project Jupyter
Tags:
- jupyter
- jupyter-notebook
- jupyterlab-desktop
Moniker: jupyterlab
```
I suggest adding tags and a moniker so that the package can be found easily and installed with `winget install jupyterlab`